flows/resumes/base.go

```diff
@@ -73,15 +73,34 @@ func (r *baseResume) Apply(run flows.FlowRun, logEvent flows.EventCallback) {
 	}
 }
 
+//------------------------------------------------------------------------------------------
+// Expressions context
+//------------------------------------------------------------------------------------------
+
+// Context is the schema of trigger objects in the context, across all types
+type Context struct {
+	type_ string
+	dial  types.XValue
+}
+
+func (c *Context) asMap() map[string]types.XValue {
```

Comment:
copies same approach we use in triggers to enforce same schema across different types